packages/chat-stateful-client/src/EventSubscriber.ts

```diff
@@ -62,7 +62,14 @@ export class EventSubscriber {
   };
 
   private onChatMessageReceived = (event: ChatMessageReceivedEvent): void => {
+    // Today we are avoiding how to render these messages. In the future we can
+    // remove this condition and handle this message appropriately.
+    if (event.type !== 'Text' && event.type !== 'RichText/Html') {
```

Comment:
Do we need a condition for 'html'? https://github.com/Azure/communication-ui-library/pull/946/files

Looking to avoid breaking: https://github.com/Azure/communication-ui-library/issues/824

Reply:
nit: convert the type we are checking to lowercase and use the enum `ACSKnownMessageType` we have for checking against

Reply:
Yea, 'html' type is coming from teams sometimes - not sure if that is also the case for notifications

Reply:
Huge inconsistence of type name/lower/upper cases from chatThreadClient.getMessages() and these notifications, which should be resolved by our chat sdk in their further release

Reply:
Yep I am going to toLowerCase() and do checks against text richtext/html and html

all other messages are currently dropped.